antarest/study/web/study_data_blueprint.py

```diff
@@ -1577,4 +1582,337 @@ def set_thermal_form_values(
+        - `form`: The characteristic of the storage that we can update:
+             - `name`: The name of the updated storage.
+             - `group`: The group of the updated storage.
+             - `injectionNominalCapacity`: The injection Nominal Capacity of the updated storage.
+             - `withdrawalNominalCapacity`: The withdrawal Nominal Capacity of the updated storage.
+             - `reservoirCapacity`:  The reservoir capacity of the updated storage.
+             - `efficiency`: The efficiency of the updated storage
+             - `initialLevel`: The initial Level of the updated storage
+             - `initialLevelOptim`: The initial Level Optim of the updated storage
+
+        Returns: The updated storage with the following attributes:
+         - `name`: The name of the updated storage.
+         - `group`: The group of the updated storage.
+         - `injectionNominalCapacity`: The injection Nominal Capacity of the updated storage.
+         - `withdrawalNominalCapacity`: The withdrawal Nominal Capacity of the updated storage.
+         - `reservoirCapacity`:  The reservoir capacity of the updated storage.
+         - `efficiency`: The efficiency of the updated storage
+         - `initialLevel`: The initial Level of the updated storage
+         - `initialLevelOptim`: The initial Level Optim of the updated storage
+         - `id`: The storage ID of the study that we want to update.
+
+        Permissions:
+        - User must have READ/WRITE permission on the study.
+        """
+
+        logger.info(
+            f"Set storage short term from {area_id} for study {uuid} by giving his id:{storage_id}",
+            extra={"user": current_user.id},
+        )
+        params = RequestParameters(user=current_user)
+        study = study_service.check_study_access(
+            uuid, StudyPermissionType.WRITE, params
+        )
+        return study_service.st_storage_manager.update_st_storage(
+            study, area_id, storage_id, form
+        )
+
+    @bp.delete(
+        path="/studies/{uuid}/areas/{area_id}/storages/{storage_id}",
+        tags=[APITag.study_data],
+        summary="Remove a short-term storage from an area",
+        status_code=HTTPStatus.NO_CONTENT,
+    )
+    def delete_st_storage(
+        uuid: str,
+        area_id: str,
+        storage_id: str,
+        current_user: JWTUser = Depends(auth.get_current_user),
+    ) -> None:
+        """
+        Delete a short-term storage from an area.
+
+        Args:
+        - `uuid`: The UUID of the study.
+        - `area_id`: The area ID.
+        - `storage_id`: The storage id of the study that we want to delete.
+
+        Returns:  An empty content
+
+        Permissions:
+        - User must have READ/WRITE permission on the study.
+        """
+        logger.info(
+            f"Delete storage short term from {area_id} for study {uuid} by giving his id:{storage_id}",
```

Comment:
"Delete short-term storage {storage_id} from {area_id} for study {uuid}"

Reply:
Done